test/functional/test_framework/wallet.py

```diff
@@ -182,6 +182,16 @@ def sendrawtransaction(self, *, from_node, tx_hex):
         from_node.sendrawtransaction(tx_hex)
         self.scan_tx(from_node.decoderawtransaction(tx_hex))
 
+    def send_large_transactions(self, node, array_of_large_tx, num_txns, fee_rate):
```

Comment:
Is this needed to be put into miniwallet?

If not, it would be better suited in the test that uses it. Either as function or inline, because it is only called once.

Regardless, the interface should be cleaned up. `array_of_large_tx` is *not* an array of txs, but of txouts.

Reply:
I thought it could make sense as a shared helper since I expected the functionality to be useful in different tests as well. Although this only introduces one caller, would make it easier for others to find.

Reply:
The issue I see is that the interface isn't correct in that the passed `fee_rate` won't be the fee rate of the tx returned. Inlining it would avoid the confusion in the interface.

Reply:
gotcha, yeah agreed that the interface should be self-evident if the function is kept here, or moving it over would be another option.

Reply:
Makes sense, this is a very specific case hence I have moved the helper function to the `mempool_limit.py`
Thanks for the suggestion